File: IOT/shakap12_ShashwatKapoor_1923ec1177_3/Patterns/Pattern2.java
import java.util.*;

public class Main {

    public static void main(String[] args) {
        // write ur code here
        Scanner sc=new Scanner(System.in);
        int n=sc.nextInt();
        for(int x=n;x>=0;x--){
            for(int y=1;y<=x;y++){
                System.out.print("*"+"\t");
            }
            System.out.println();
        }


    }
}

File: ML/Coderaman-tech_AmanKumarShrivastava_2024cse1155_2/Patterns/Pattern2.cpp
#include <iostream>
using namespace std;

int main(int argc, char **argv){
    int n;
    cin >> n;
    int i,j;
    for(i=n;i>=1;i--){
        for(j=1;j<=i;j++){
            cout<<"*\t";
        }
        cout<<"\n";
    }
    //write your code here
    
}